library/Reporting/Hook/EmailProviderHook.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Icinga\Module\Reporting\Hook;
+
+use Icinga\Application\Hook;
+
+abstract class EmailProviderHook
+{
+    /**
+     * Get all Contact eMails
+     *
+     * @return mixed
+     */
+    abstract public function getContactEmails();
```

Comment:
I think this should return an array as well. After all, you're using this method in the `SendForm` as if it were returning an array and not a mixed one.